features/list.feature

```diff
@@ -0,0 +1,21 @@
+Feature: List
+  As a reader
+  I want to read posts from my favorite blogger
+  So that I can learn something
+  
+  Scenario: readers accessing published posts
+    Given I am on the post submission page again
+    When I create a valid post
+    Then the post is immediately available to readers on the homepage
+    
+  Scenario: viewing post title
+    When the post title is visible
+    Then the title is not null
```

Comment:
The two steps to this scenario are redundant. This also seems like an unnecessary test given the acceptance criteria.